Apps/W1/Shopify/app/src/Logs/Pages/ShpfySkippedRecords.Page.al

```diff
@@ -56,6 +66,30 @@ page 30166 "Shpfy Skipped Records"
                     Rec.ShowPage();
                 end;
             }
+            action(Delete7days)
+            {
+                ApplicationArea = All;
+                Caption = 'Delete Entries Older Than 7 Days';
+                Image = ClearLog;
+                ToolTip = 'Clear the list of log entries that are older than 7 days.';
+
+                trigger OnAction();
+                begin
+                    Rec.DeleteEntries(7);
+                end;
+            }
+            action(Delete0days)
+            {
+                ApplicationArea = All;
+                Caption = 'Delete All Entries';
+                Image = Delete;
+                ToolTip = 'Clear the list of all log entries.';
```

Comment:
I think this from copy paste. We should make the tooltip meaningful, not log entries but skipped record entries